<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartWidget</class>
 <widget class="QWidget" name="StartWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="learnButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>LEARN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="quizButton">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>400</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>QUIZ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resultsButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>RESULTS</string>
   </property>
  </widget>
  <widget class="Physics" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="pooedPoo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-50</y>
      <width>531</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="replayButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Again!</string>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>learnButton</zorder>
  <zorder>quizButton</zorder>
  <zorder>resultsButton</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Physics</class>
   <extends>QWidget</extends>
   <header location="global">physics.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>learnButton</tabstop>
  <tabstop>quizButton</tabstop>
  <tabstop>resultsButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
